manifest-rhel-9.2.yaml

```diff
@@ -150,3 +142,6 @@ repo-packages:
       - cri-tools
       - openshift-clients
       - openshift-hyperkube
+  - repo: rhel-9.0-fast-datapath
```

Comment:
Aside: I've been thinking for a while we should support specifying repos here that aren't listed in the toplevel `repos` key. The semantic would be that we enable this repo only for the specified packages and nothing else.